colossalai/nn/optimizer/cpu_adam.py

```diff
@@ -31,18 +31,46 @@ def __init__(self,
         except ImportError:
             raise ImportError('Please install colossalai from source code to use CPUAdam')
         self.cpu_adam_op = cpu_adam
-        self.cpu_adam_op.create_adam(self.opt_id,
-                                     lr,
-                                     betas[0],
-                                     betas[1],
-                                     eps,
-                                     weight_decay,
-                                     adamw_mode,
-                                     simd_log)
+        self.cpu_adam_op.create_adam(self.opt_id, lr, betas[0], betas[1], eps, weight_decay, adamw_mode, simd_log)
 
     def __del__(self):
         self.cpu_adam_op.destroy_adam(self.opt_id)
 
+    def torch_adam_update(self,
```

Comment:
Can we just use original `torch.optim.adam.adam` ?

Reply:
I try to import the function, it failed
```
from torch.optim._functional.adam
```
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'torch.optim._functional.adam'; 'torch.optim._functional' is not a package
https://github.com/pytorch/pytorch/blob/0fc6bd2e47/torch/optim/_functional.py#L54

Reply:
OK. I see.